articles/service-fabric/service-fabric-get-started-containers.md

```diff
@@ -327,6 +327,56 @@ NtTvlzhk11LIlae/5kjPv95r3lw6DHmV4kXLwiCNlcWPYIWBGIuspwyG+28EWSrHmN7Dt2WqEWqeNQ==
 </ServiceManifestImport>
 ```
 
+### Configure cluster-wide credentials
+
+Starting v6.3, Service Fabric allow user to configure cluster-wide credentials which can be used as default repository credentials by applications.
+
+You can enable/disable the feature by adding a "UseDefaultRepositoryCredentials" attribute to ContainerHostPolicies in ApplicationManifest.xml with a "true/false" Boolean value.
+
+```xml
+<ServiceManifestImport>
+    ...
+    <Policies>
+        <ContainerHostPolicies CodePackageRef="Code" UseDefaultRepositoryCredentials="true">
+            <PortBinding ContainerPort="80" EndpointRef="Guest1TypeEndpoint"/>
+        </ContainerHostPolicies>
+    </Policies>
+    ...
+</ServiceManifestImport>
+```
+
+This will tell Service Fabric to use the default repository credentials which you can specify in the ClusterManifest under the hosting section.  If UseDefaultRepositoryCredentials is set to true, service fabric will now read the following values in clustermanifest:
+
+* DefaultContainerRepositoryAccountName (string)
+* DefaultContainerRepositoryPassword (string)
+* IsDefaultContainerRepositoryPasswordEncrypted (bool)
+
+Here is an example of what you can add inside the Hosting section in ClusterManifestTemplate.json
+
+```json
+      {
+        "name": "Hosting",
+        "parameters": [
+          {
+            "name": "EndpointProviderEnabled",
+            "value": "true"
+          },
+          {
+            "name": "DefaultContainerRepositoryAccountName",
+            "value": "someusername"
+          },
+          {
+            "name": "DefaultContainerRepositoryPassword",

```

Comment:
Can this be an encrypted password?

Reply:
Yes by setting IsDefaultContainerRepositoryPasswordEncrypted to true.

Reply:
How do you do this? We need to specify that in this document.